src/examples/java/com/nylas/examples/other/RoomResourceExample.java

```diff
@@ -0,0 +1,23 @@
+package com.nylas.examples.other;
+
+import com.nylas.NylasAccount;
+import com.nylas.NylasClient;
+import com.nylas.RoomResource;
+import com.nylas.RoomResources;
+import com.nylas.examples.ExampleConf;
+
+import java.util.List;
+
+public class RoomResourceExample {
+    public static void main(String[] args) throws Exception {
+        ExampleConf conf = new ExampleConf();
+        NylasClient client = new NylasClient();
+        NylasAccount account = client.account(conf.get("access.token"));
+
+        RoomResources roomResource = account.roomResources();
+        List<RoomResource> roomResourceList = roomResource.list().fetchAll();
+        for(RoomResource resource : roomResourceList) {
+            System.out.println(resource);
+        }
```

Comment:
I don't have a GSuite or Office 365 account to test this out.  Have you been able to?

Reply:
Yes I have! I have a GSuite org and I was able to test it out.